Program.cs

```diff
@@ -629,57 +654,93 @@ public void itemShop()
             if (hasPlayerBeenInShop == true)
             {
                 somethingCool();
+                if (playerHasWatch == false)
+                {
+                    GetWatch(totalXP);
+                }
```

Comment:
On the second entry into the shop, we may get both the SOS clue and the watch... do we want that. But perhaps we are safe because of the XP limit required. 

Perhaps we can look to randomly select clues from a list to ensure we only ever get a max of 1 in any one visit.